### PR TITLE
Saas 11.5 web hide remove btn pgu

### PR DESCRIPTION
**Pull request by pgu-odoo**
Description of the issue/feature this PR addresses:
task id -
https://www.odoo.com/web#id=1886913&action=327&model=project.task&view_type=form&menu_id=4720
Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/fields/relational_fields.js [hunk 1]
```diff
@@ -1540,6 +1541,7 @@ var FieldOne2Many = FieldX2Many.extend({
                 self._setValue({operation: 'DELETE', ids: [id]});
             },
             deletable: this.activeActions.delete,
```
Comment (ged-odoo) [inline addons/web/static/src/js/fields/relational_fields.js]: deletable: this.activeActions.delete && ev.data.showRemove,

File: addons/web/static/src/js/fields/relational_fields.js [hunk 2]
```diff
@@ -1540,6 +1541,7 @@ var FieldOne2Many = FieldX2Many.extend({
...
+            showRemove: ev.data.showRemove,
```
Comment (ged-odoo) [inline addons/web/static/src/js/fields/relational_fields.js]: remove this

**Comment by pgu-odoo** [comment]
duplicate to https://github.com/odoo/odoo/pull/28792

File: addons/web/static/src/js/views/view_dialogs.js [hunk 3]
```diff
@@ -113,6 +115,7 @@ var FormViewDialog = ViewDialog.extend({
         this.shouldSaveLocally = options.shouldSaveLocally;
         this.readonly = options.readonly;
         this.deletable = options.deletable;
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/view_dialogs.js]: maybe you could have this.deletable = options.deletable && this.showRemove; and then, not using showRemove elsewhere in this file

**Pull request closed without merging**